### ISSUE TITLE
"No tC default tools found." Error when launching project with alignment tool

### ISSUE THREAD

**Issue by jag3773** [unfoldingWord/translationCore#3498]
Affects at least Mac and Linux builds.

Steps to reproduce:

1. Launch translationCore 0.8.1-rc.3-d8d4486 from http://tc-sandbox.unfoldingword.surge.sh/
2. Load https://git.door43.org/aniljohn/hi_tit or https://git.door43.org/katalakay/English_tit
3. Launch WordAlignment tool.
4. See error:
![image](https://user-images.githubusercontent.com/107731/34361119-5e7c26d2-ea35-11e7-8447-38fa90b544fa.png)

Console error text:
```
Container.js? [sm]:14 Uncaught (in promise) TypeError: Cannot read property 'currentPaneSettings' of undefined
    at Container.componentWillMount (Container.js? [sm]:2)
    at /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:348
    at measureLifeCyclePerf (/home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:75)
    at ReactCompositeComponentWrapper.performInitialMount (/home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:347)
    at ReactCompositeComponentWrapper.mountComponent (/home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:258)
    at Object.mountComponent (/home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactRec…:46)
    at ReactCompositeComponentWrapper.performInitialMount (/home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:371)
    at ReactCompositeComponentWrapper.mountComponent (/home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:258)
    at Object.mountComponent (/home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactRec…:46)
    at ReactCompositeComponentWrapper.performInitialMount (/home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:371)
componentWillMount @ Container.js? [sm]:14
(anonymous) @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:348
measureLifeCyclePerf @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:75
performInitialMount @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:347
mountComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:258
mountComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactRec…:46
performInitialMount @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:371
mountComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:258
mountComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactRec…:46
performInitialMount @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:371
mountComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:258
mountComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactRec…:46
performInitialMount @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:371
mountComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:258
mountComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactRec…:46
mountChildren @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactMul…:238
_createInitialChildren @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactDOM…:697
mountComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactDOM…:516
mountComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactRec…:46
mountChildren @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactMul…:238
_createInitialChildren @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactDOM…:697
mountComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactDOM…:516
mountComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactRec…:46
updateChildren @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactChi…:121
_reconcilerUpdateChildren @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactMul…:208
_updateChildren @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactMul…:312
updateChildren @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactMul…:299
_updateDOMChildren @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactDOM…:936
updateComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactDOM…:754
receiveComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactDOM…:716
receiveComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactRec…:125
_updateRenderedComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:754
_performComponentUpdate @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:724
updateComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:645
receiveComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:547
receiveComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactRec…:125
_updateRenderedComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:754
_performComponentUpdate @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:724
updateComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:645
receiveComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:547
receiveComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactRec…:125
updateChildren @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactChi…:109
_reconcilerUpdateChildren @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactMul…:208
_updateChildren @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactMul…:312
updateChildren @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactMul…:299
_updateDOMChildren @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactDOM…:936
updateComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactDOM…:754
receiveComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactDOM…:716
receiveComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactRec…:125
_updateRenderedComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:754
_performComponentUpdate @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:724
updateComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:645
receiveComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:547
receiveComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactRec…:125
updateChildren @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactChi…:109
_reconcilerUpdateChildren @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactMul…:208
_updateChildren @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactMul…:312
updateChildren @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactMul…:299
_updateDOMChildren @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactDOM…:936
updateComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactDOM…:754
receiveComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactDOM…:716
receiveComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactRec…:125
_updateRenderedComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:754
_performComponentUpdate @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:724
updateComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:645
receiveComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:547
receiveComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactRec…:125
_updateRenderedComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:754
_performComponentUpdate @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:724
updateComponent @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:645
performUpdateIfNecessary @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactCom…:561
performUpdateIfNecessary @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactRec…:157
runBatchedUpdates @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactUpd…:150
perform @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/Transact…:140
perform @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/Transact…:140
perform @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactUpd…:89
flushBatchedUpdates @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactUpd…:172
closeAll @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/Transact…:206
perform @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/Transact…:153
batchedUpdates @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactDef…:62
enqueueUpdate @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactUpd…:200
enqueueUpdate @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactUpd…:24
enqueueSetState @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-dom/lib/ReactUpd…:209
ReactComponent.setState @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react/lib/ReactCompone…:63
onStateChange @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/react-redux/lib/compon…:222
dispatch @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/redux/lib/createStore.…:186
(anonymous) @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/redux-thunk/lib/index.…:14
dispatch @ /home/jesse/Downloads/translationCore8-1-rc3/resources/app.asar/node_modules/redux/lib/applyMiddlew…:45
(anonymous) @ ProjectDataLoadingActions.js? [sm]:33
```

**Comment by cckozie** [comment]
This is not a problem in the latest Develop build (0.8.1-alpha.13-17f8690) or Sandbox build (0.8.1-rc.3-d8d4486) on either Windows or Mac. At least I'm not seeing it.

**Comment by jag3773** [comment]
I'm also not able to reproduce on Mac.

**Comment by cckozie** [comment]
0.8.1-rc.4-215318b
I now have this problem on my Mac. I started with an empty projects folder, installed the latest build, imported a project from door43, and selected the Word Alignment tool.
Same thing happens when importing a new project as well as loading the one that was previously imported. Doesn't matter is project has alignment data or not.
![image](https://user-images.githubusercontent.com/8171759/34389211-06673ce2-eafe-11e7-8ead-dfe4731e30dd.png)

**Comment by cckozie** [comment]
Need to retest on clean Windows and Mac machines.

**Comment by jag3773** [comment]
@cckozie Problem is solved on my Linux machine by Commit 606bcf5 on release-0.8.1, Travis build #5724.

**Comment by cckozie** [comment]
Problem is fixed in 0.8.1-rc.7-8b741cd for both Windows 64bit and Mac.
